src/search.c

```diff
@@ -167,7 +170,10 @@ static void print_final_results(void)
 		b = ptr->data;
 		ptr = ptr->next;
 		counter++;
-		printf("Bundle %s   (%li Mb)\n", b->bundle_name, b->size);
+		printf("Bundle %s\t(%li Mb)\t%s\n",
+		       b->bundle_name,
+		       b->size,
+		       b->is_tracked ? "installed" : "");
```

Comment:
Maybe 

		printf("Bundle %s\t(%li Mb)%s\n",
		       b->bundle_name,
		       b->size,
		       b->is_tracked ? "\tinstalled" : "");

to avoid the trailing tab?  Maybe
		printf("Bundle %s\t(%li Mb)\t%sinstalled\n",
		       b->bundle_name,
		       b->size,
		       b->is_tracked ? "" : "not-");

to keep the number of fields the same.

Reply:
Oooh, I like the first suggestion. Makes it obvious that this one is different. The second one you have to look at a little closer to see the `not-` in front of `installed`